website_hr_job_unpublication/data/ir_cron.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
```

Comment:
For a cron, you have to put noupdate="True" otherwise the nextcall value will be updated each time you update the module (and the cron will be active = True even if a user disable it).